<div class="row">
  <div class="columns medium-12">
    <h2>Data Content</h2>
    <div class="medium-12">
      <h3>
        What can be described as a complex
      </h3>
      <p>
        A stable set of (two or more) interacting macromolecules such as proteins which can be co-purified by an
        acceptable method and have been shown to exist as an isolated, functional unit in vivo. Any interacting,
        integral non-protein molecules (e.g. small molecules, nucleic acids) may also be included.
      </p>
    </div>

    <div class="columns medium-6">
      <h3>What should <b>NOT</b> be captured:</h3>
      <ul>
        <li>
          Enzyme/substrate, receptor/ligand or any similar transient interactions unless these are a critical part of
          the complex assembly (e.g. PDGF receptors only become 'dimeric' when linked by the dimeric ligand forming a
          tetramer).
        </li>
        <li>
          Proteins associated in a pulldown/coimmunoprecipitation with no functional link or any evidence that this is
          a defined biological entity rather than a loose affinity complex.
        </li>
        <li>
          Any literature complex where the only evidence is based on genetic interaction data.
        </li>
        <li>
          Partial complexes
        </li>
      </ul>
    </div>
    <div class="columns medium-6">
      <h3>Tricky cases we <b>DO</b> capture:</h3>
      <ul>
        <li>
          Substrates or ligands if the enzyme or receptor complex only forms in their presence (see PDGF receptors
          above, e.g. <a (click)="goToComplex('EBI-9082861')">EBI-9082861</a>).
        </li>
        <li>
          Homologous proteins, with the same functionality, which would be inferred based on homology of the
          genome-encoded components made primarily on functional conservation between the two systems to form a
          complex but for which no physical link has been demonstrated, e.g. proteins A and B have been shown to
          physically interact and form a functional complex, protein C is a homologue of protein B by sequence
          similarity and is know to have the same function as B but protein A-C interaction has not been demonstrated
          experimentally (E.g. SUMO - E1 ligase complexes where there is interaction evidence for binding with SUMO1
          (<a (click)="goToComplex('EBI-9349603')">EBI-9349603</a>) but not with
          SUMO2 (<a (click)="goToComplex('EBI-9345927')">EBI-9345927</a>)). These
          complexes are tagged with <a
          href="http://www.ebi.ac.uk/ols/ontologies/eco/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FECO_0005547"
          target="_blank">ECO:0005547 <i class="icon icon-generic small" data-icon="x"></i></a> biological system
          reconstruction evidence based on inference from background scientific knowledge used in manual assertion.
        </li>
        <li>
          Complexes that lack full experimental evidence but are commonly regarded as existing, e.g. transmembrane
          receptors (e.g <a (click)="goToComplex('EBI-9008426')">EBI-9008426</a>,
          GABA receptors) for which only pharmacological evidence exists. These complexes
          are tagged with <a (click)="goToECO('ECO:0000306')">ECO:0000306 <i class="icon icon-generic small" data-icon="x"></i></a>
          - inferred from background scientific knowledge by manual assertion.
        </li>
      </ul>
    </div>
    <div class="medium-12">
      <h3>Complex Nomenclature</h3>
      <dl>
        <dt>
          <b> Complex recommended name</b>
        </dt>
        <dd>
          The most informative, well accepted name in the literature, that is intuitive to the user. Where possible,
          this will be the same as the equivalent component term in GO. The term should always end in the word 'complex'
          or homo'n'mer. The recommended name is kept consistent when a complex is conserved across a taxonomic range.
        </dd>
      </dl>
      <dl>
        <dt><b>Complex systematic name</b></dt>
        <dd>Derived using the Reactome rules for naming complexes. In principle this is a string of gene names of the
          participants of the complex separated by a colon (e.g. "fiba:fibb:fibg"). The order is determined by the order
          of synthesis of the complex. Where several participants join at the same time, or the order is unknown,
          alphanumeric order is used. Recommended gene names for each model organism will be used; therefore the
          systematic name may not be consistent when a complex is conserved across a taxonomic range.
        </dd>
      </dl>
      <dl>
        <dt><b>Complex synonym</b></dt>
        <dd>All other possible names the complex is known by or can be described as.</dd>
      </dl>
      <dl>
        <dt><b>Short label</b></dt>
        <dd>Currently an obligate part of the data model, it may be possible to remove these once the model is updated.
          Currently, an appropriate designation for the complex with species indicated using the UniProt five letter
          code e.g. fibrinogen_human, tfiid_mouse. When the same complex is conserved across a taxonomic range, the root
          name is maintained across all entries e.g. fibrinogen_human, fibrinogen_mouse, fibrinogen_bovin.
        </dd>
      </dl>
    </div>
    <div class="medium-12">
      <h3>Complex Participants</h3>
      <dl>
        <dt><b>Proteins</b></dt>
        <dd>All proteins are derived from, and linked to, UniProtKB, ideally UniProtKB/Swiss-Prot.
          Isoform and chain designators will be used when appropriate. Should one or more isoforms exist, annotation
          will
          be to the canonical protein entry unless either only one isoform is known to exist in the complex or different
          isoforms give the complex different properties. In the latter case, a separate entry should be made for each
          variation with detail given in "curated-complex" or "complex-properties" as appropriate (see below for details
          on complex variants).
        </dd>
      </dl>
      <dl>
        <dt><b>Small molecules /polysaccharides</b></dt>
        <dd>All small molecules are derived from, and linked to, ChEBI. Small
          molecules are entered if they are integral to the complex or bind to the complex as part of its function, e.g.
          cofactors, electron donors/acceptors, such as ATP, H+. Enzyme targets are not added as participants. For
          example, ATP is entered as a cofactor if the enzyme function is NOT primarily an ATPase (e.g. <a (click)="goToComplex('EBI-9008779')">EBI-9008779</a>
          gyrase_ecoli) but NOT entered for ATPases where it is a substrate (e.g. <a (click)="goToComplex('EBI-9007893')">EBI-9007893</a> mfd-uvra_ecoli, a DNA
          translocase).
        </dd>
      </dl>
      <dl>
        <dt><b>Nucleic acids</b></dt>
        <dd>Nucleic acids are only entered as participants when they are an obligate part of the
          complex. Non-coding RNA participants are linked to RNACentral, all other nucleic acid types are created as
          generic molecules linked to generic ChEBI identifiers. For complexes which assemble and then bind to a nucleic
          acid, this function is indicated in free text and using GO terms such as GO:0003677 DNA binding.
        </dd>
      </dl>
    </div>
    <div class="medium-12">
      <h3>Participant Features</h3>
      <ul>
        <li>Any features known to be involved in the reactions are mapped to the underlying sequence, as given in the
          source database, and cross-referenced to InterPro when possible. If a PTM is required for complex activation,
          this is curated as a feature and the effects detailed in the annotation field 'Complex-properties'.
        </li>
        <li>Binding sites or residues within proteins, known to directly interact within the complex, are shown as
          linked features in the graphical views.
        </li>
        <li>Stoichiometry is added, when known. Stoichiometry=0 is used for participants with no information about
          stoichiometry. It is common that stoichiometry is only know for some participants in the same complex.
        </li>
      </ul>
    </div>
    <div class="medium-12">
      <h3>Interaction Type</h3>
      <p>
        This will always be 'Physical association' - the controlled vocabulary term indicating that these proteins are
        present in the same complex - unless there are one or two protein species involved in which case it will be
        'Direct interaction'. Proteins directly binding to each other within a larger complex will be indicated by
        linked features (see above).
      </p>
    </div>
    <div class="medium-12">
      <h3>Free text annotation</h3>
      <dl>
        <dt><b>Curated-complex</b></dt>
        <dd>A brief, free-text description of the function of the complex, written in the same
          style as a UniProtKB/Swiss-Prot entry. For example "Required for processive DNA replication and may act as a
          replicative helicase during DNA synthesis. Plays a central role in S-phase genome stability.".
        </dd>
      </dl>
      <dl>
        <dt><b>Complex-properties</b></dt>
        <dd>Details of physical properties of the complex. This may include details about the
          topology, varying (as opposed to absolute) stoichiometry, molecular weight and Stoke's radius of the complex.
        </dd>
      </dl>
      <dl>
        <dt><b>Complex-assembly</b></dt>
        <dd>Experimentally verified structural assembly e.g. homodimer, heterohexamer. Assemblies
          which have been computationally predicted are not included.
        </dd>
      </dl>
      <dl>
        <dt><b>Disease</b></dt>
        <dd>Only added when the disease state has been specifically linked to the protein when in the
          complex.
        </dd>
      </dl>
      <dl>
        <dt><b>Ligand</b></dt>
        <dd>When a complex has one or more ligands transiently associated with it, they will be listed
          under this heading.
        </dd>
      </dl>
    </div>
    <div class="medium-12">
      <h3>Structured Annotations</h3>
      <p>All structured annotation is entered as database/controlled vocabulary cross-reference with an appropriate
        qualifier term.</p>
      <dl>
        <dt><b>Gene Ontology</b></dt>
        <dd>Used to indicate the function, process and component of the complex as a whole. The
          Function term, in particular, may not be true for all members of the complex, for example enzyme complexes
          will
          be annotated with a catalytic function term even when some subunits play only a regulatory role.
        </dd>
      </dl>
      <dl>
        <dt><b>Experimental evidence</b></dt>
        <dd>When high quality evidence for the existence of this complex is present in an
          IMEx database, this will be added manually as a cross-reference so that it may be downloaded in the same file
          as
          the complex.
        </dd>
      </dl>
      <dl>
        <dt><b>3D structure</b></dt>
        <dd>Representative PDB cross-references will be added when the complex has been crystallised
          in its entirety.
        </dd>
      </dl>
      <dl>
        <dt><b>Electron microscopy</b></dt>
        <dd>Representative EMDB cross-references will be added when the complex has been
          visualised by electron microscopy in its entirety.
        </dd>
      </dl>
      <dl>
        <dt><b>Evidence codes</b></dt>
        <dd>The following ECO codes will be used to indicate the strength of evidence that a
          complex exists:
        </dd>
        <table class="hover">
          <thead>
          <tr>
            <th width="150">ECO Code</th>
            <th width="300">Name</th>
            <th>Description</th>
          </tr>
          </thead>
          <tbody>
          <tr>
            <td><a (click)="goToECO('ECO:0005542')">ECO:0005542 <i class="icon icon-generic small" data-icon="x"></i></a></td>
            <td>biological system reconstruction evidence by experimental evidence from single species used
              in manual assertion
            </td>
            <td>Indicates that experimental evidence for the complexes exists but only by combining data
              from several experiments. Experimental data can be found in an IMEx database, PDB or EMDB.
            </td>
          </tr>
          <tr>
            <td><a (click)="goToECO('ECO:0000353')">ECO:0000353 <i class="icon icon-generic small" data-icon="x"></i></a></td>
            <td>physical interaction evidence used in manual assertion</td>
            <td>Indicates that experimental evidence for the complexes exists in a single experiment. This will consist
              of either a cross-reference to experimental data in an IMEx database, PDB or EMDB.
            </td>
          </tr>
          <tr>
            <td><a (click)="goToECO('ECO:0005543')">ECO:0005543 <i class="icon icon-generic small" data-icon="x"></i></a></td>
            <td>biological system reconstruction evidence by experimental evidence from mixed species used in
              manual assertion
            </td>
            <td>Indicates that experimental evidence for the complexes exists in a single experiment but the constructs
              are derived from homologous gene products in different species. Experimental data can be found in an IMEx
              database, PDB or EMDB.
            </td>
          </tr>
          <tr>
            <td><a (click)="goToECO('ECO:0005544')">ECO:0005544 <i class="icon icon-generic small" data-icon="x"></i></a></td>
            <td>biological system reconstruction evidence based on orthology evidence used in manual assertion</td>
            <td>Indicates only limited experimental evidence exists for a complex in one species (e.g. mouse) but it is
              desirable to curate the complex which has been curated in another species (e.g. human). Sequences and
              number
              of genome-encoded components are fairly conserved but some divergence may be observed. The complex with
              the
              experimental evidence has to be cross-referenced with the qualifier = "inferred-from" and must have the
              ECO:0000353, ECO:0005542 or ECO:0005543 tag.
            </td>
          </tr>
          <tr>
            <td><a (click)="goToECO('ECO:0005546')">ECO:0005546 <i class="icon icon-generic small" data-icon="x"></i></a></td>
            <td>biological system reconstruction evidence based on paralogy evidence used in manual assertion</td>
            <td>Indicates only limited experimental evidence exists for one complex but full experimental evidence
              exists for a similar complex of the same species. Sequences and number of genome-encoded components are
              fairly conserved but some divergence may be observed. The complex with the experimental evidence has to be
              cross-referenced with the qualifier = "inferred-from" and must have the ECO:0000353, ECO:0005542 or
              ECO:0005543 tag.
            </td>
          </tr>
          <tr>
            <td><a (click)="goToECO('ECO:0005547')">ECO:0005547 <i class="icon icon-generic small" data-icon="x"></i></a></td>
            <td>biological system reconstruction evidence based on inference from background scientific
              knowledge used in manual assertion
            </td>
            <td>Is used if no or only partial experimental evidence is present but the
              complex is generally assumed to exist. Functional studies or ligand binding evidence from pharmacological
              experiments are often used for the reconstruction of such complexes.
            </td>
          </tr>
          </tbody>
        </table>
      </dl>
      <dl>
        <dt><b>Enzymatic activity</b></dt>
        <dd>The E.C. number linked to IntEnz will be added when an enzyme complex is described.</dd>
      </dl>
      <dl>
        <dt><b>Additional literature</b></dt>
        <dd>Review articles or experimental data not appropriate for entering into IntAct
          are added.
        </dd>
      </dl>
      <dl>
        <dt><b>Pathway information</b></dt>
        <dd>For human complexes, crosslinks to Reactome put complexes into a pathway context.
          Note that the definition of a complex is different in Reactome and in many cases a one-to-many relationship
          exists.
        </dd>
      </dl>
      <dl>
        <dt><b>Disease information</b></dt>
        <dd>Cross references to the Experimental Factor Ontology (EFO) or their contributing databases (e.g. Orphanet,
          Human Phenotype Ontology) may be added if a complex or a chain when within that complex has been linked to a
          specific disease condition.
        </dd>
      </dl>
      <dl>
        <dt><b>Drug target information</b></dt>
        <dd>Cross-links to ChEMBL are used to indicate complexes which have been used as
          drug targets.
        </dd>
      </dl>
    </div>
    <div class="medium-12">
      <h3>Complex Variants</h3>
      <p>
        If variant forms of a complex exist i.e. the same functional unit can exist in alternate forms with differing
        macromolecular composition, these are curated as separate objects. For example, PDGF can exist as a PDGF-A
        homodimer, PDGF-B homodimer, PDGF-AB heterodimer, PDGF-C homodimer and a PDGF-D homodimer If the variants have
        well-accepted names, e.g. PDGF-AB, these may be used as the primary name. If not, then the recommended name is
        qualified by variant 1, variant 2 e.g. TRAMP complex variant 1 (<a (click)="goToComplex('EBI-2352894')">EBI-2352894</a>).
      </p>
    </div>
  </div>
</div>
